public/sw.js

```diff
@@ -0,0 +1,39 @@
+const CACHE_NAME = "devJotPWA-v1";
+const CACHE_FILE = ["./index.html", "./js/index.js", "./css/style.css"];
```

Comment:
I just tested this out locally. Btw (I'm not sure if you knew this already) you can test the offline experience by opening Chrome dev tools & selecting 'offline' from the throttling dropdown:

![Screen Shot 2020-12-17 at 11 44 28](https://user-images.githubusercontent.com/30320536/102483886-56f58800-405d-11eb-92cf-619faad5d10e.png)

If you empty the cache and reload we get some errors – looks like we just need to add `"https://cdn.quilljs.com/1.3.6/quill.js"` & `"https://cdn.quilljs.com/1.3.6/quill.snow.css"` to the `CACHE_FILE` array.